Drivers/CAN/can.c

```diff
@@ -0,0 +1,71 @@
+/*
+    CAN DRIVER Source File
+    Link to part Datasheet for reference:
+    https://www.st.com/resource/en/user_manual/um1725-description-of-stm32f4-hal-and-lowlayer-drivers-stmicroelectronics.pdf
+    
+    Author: Hamza Iqbal
+*/
+
+#include "can.h"
+
+
+void CAN_Handler_Init(CAN_HandleTypeDef* CAN_Line)
```

Comment:
Not sure if this implementation needs to be specific to the SN65HVD CAN transceiver we are using, if it is we should rename these functions to ```sn65hvd``` but it could also be a universal interface abstracted by HAL so we'll see